views/matrixsplit.html

```diff
@@ -233,8 +233,8 @@
             =
             \begin{bmatrix}
```

Comment:
I think it's because of how Mathematica simplified the operation:

![screenshot 23](https://cloud.githubusercontent.com/assets/177243/9587084/6db93dda-4fd5-11e5-81c7-c7041c7367fd.png)

The `(-1+z)` compounds were easier to understand as `(z-1)` so I homogenised the result to a "z and numerical" form instead of the other way around.